tietokanta/src/main/resources/db/migration/V1_1302__.sql

```diff
@@ -0,0 +1,112 @@
+-- ---------------------------------------------------------------------
+-- Tehtävä- ja määrätoteumiin perustuvat tavoitehintamuutokset
+-- ---------------------------------------------------------------------
+
+CREATE TYPE muutos_yksikkohinta_lahde_enum AS ENUM ('laskettu','aseta','manuaali');
+
+
+CREATE TABLE mhu_muutos_tehtava_tiedot (
+
+  versio INTEGER NOT NULL DEFAULT 1,  -- Versioinnilla oma taulu, ei liity mhu_muutos tauluun   (nämä eivät ole varsinaisia muutoksia)
+  validi_aikana TSTZRANGE NOT NULL DEFAULT TSTZRANGE(CURRENT_TIMESTAMP, NULL), -- Kertoo koska versio ollut aktiivinen 
```

Comment:
Loppuajan pitää olla 'infinity'

```suggestion
  validi_aikana TSTZRANGE NOT NULL DEFAULT TSTZRANGE(CURRENT_TIMESTAMP, 'infinity'::timestamp with time zone), -- Kertoo koska versio ollut aktiivinen 
```